modular_stonehedge/licensed-death-rattler/Death-Rattler/code/Death-Rattler.dm

```diff
@@ -0,0 +1,158 @@
+// This code is part of Death-Rattler
+// Copyright (C) 2024 Moribund/Luctus-Comptus
+// Unauthorized use, modification, or distribution of this code is prohibited without copyright holders approval. See LICENSE.md
+
+// I hate doing this. basically if you are anyone but ratwood/gyran DM me first and you can use whatever the fuck your heart wants from this. not only did he brag about theft and how easy it is he tryed to pull this same shit.
```

Comment:
(this is also not enforceable under the agplv3 but as long as you don't try to enforce it you can make threats like this all you want without violating the license)